modules/hardware/nvidia-jetson-orin.nix

```diff
@@ -10,8 +10,32 @@
 
   nixpkgs.hostPlatform.system = "aarch64-linux";
 
-  boot.loader = {
-    efi.canTouchEfiVariables = true;
-    systemd-boot.enable = true;
+  boot.kernelPatches = [
+    {
+      name = "passthrough-patch";
+      patch = ./pci-passthrough-test.patch;
+    }
+  ];
+
+  hardware.deviceTree = {
+    enable = true;
+    name = "tegra234-p3701-host-passthrough.dtb";
+  };
+
+  # Passthrough Jetson Orin WiFi card
+  boot.kernelParams = [
+    "vfio-pci" "ids=10ec:c82f"
+    "vfio_iommu_type1.allow_unsafe_interrupts=1"
```

Comment:
`allow_unsafe_interrupts` enables threat vector via MSI (message signaled interrupts). This is possible with PCI devices if there's access to device configuration space. See https://invisiblethingslab.com/resources/2011/Software%20Attacks%20on%20Intel%20VT-d.pdf - pages 6-7. 

I'll add this note to acknowledge the threat from within the netvm and make a ghaf issue out of this to document it - at least until we have iommu group interrupt remapping support or other documented VMM mitigations. It may be that there's other mitigations these days that I'm not aware of. If that turns out to be the case, let's document those and close the issue after the fact.

Reply:
I think we already had some discussion about this, and in the discussion the conclusion was that this allow_unsafe_interrupts is not needed. But I'm not sure did anyone test with this line removed